src/styles.scss

```diff
@@ -0,0 +1,54 @@
+/* You can add global styles to this file, and also import other style files */
+
+
+// Font for H1
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,opsz,wght@0,6..12,300;0,6..12,400;0,6..12,500;0,6..12,600;0,6..12,700;0,6..12,800;0,6..12,900;0,6..12,1000;1,6..12,200;1,6..12,300;1,6..12,400;1,6..12,500;1,6..12,600;1,6..12,700;1,6..12,800;1,6..12,900;1,6..12,1000&display=swap');
```

Comment:
Very cool :)

What happens if this font isn't available?

Reply:
Got it. Adding fallbacks.
        font-family: 'Nunito Sans', Arial, sans-serif;